Loader.cs

```diff
@@ -83,8 +83,15 @@ public static void OnLoad()
             new GameObject("SongCore Loader").AddComponent<Loader>();
         }
 
+        private string _customWIPPath;
+        private string _customLevelsPath;
+
         private void Awake()
         {
+            _customWIPPath = Path.Combine(Application.dataPath, "CustomWIPLevels");
+            _customLevelsPath = Path.GetFullPath(CustomLevelPathHelper.customLevelsDirectoryPath);
+            _ = BeatmapLevelsModelSO;
```

Comment:
Assigning only to be discarded again seems unnecessary to me?

Reply:
Oh yeah, I was experimenting with needing to caching that. The property `get` does a Resources.FindObjectsOfTypeAll call so i didn't want that potentially running on another thread, hence the discard to cache the backing field.

Reply:
![image](https://user-images.githubusercontent.com/41306347/147034191-ab202146-6180-4f24-858d-5f18d09fbe60.png)

Yep, it's necessary.

Reply:
Ah, that wasn't clear to me. Might wanna add a tiny comment to it to clarify that it is needed. 🙂